stable/datadog/values.yaml

```diff
@@ -534,6 +534,53 @@ clusterAgent:
   #
   createPodDisruptionBudget: false
 
+  ## @param admissionController - object - required
+  ## Admission controller webhooks that inject
+  ## cluster-wide environment variables.
+  ## Injected variables can ease setup burden when
+  ## deploying applications that require access to the agent.
+  ##
+  ## ref: https://docs.datadoghq.com/agent/cluster_agent/ TODO: link to admission controller-specific section
+  #
+  admissionController:
+    ## @param enabled - boolean - required
+    ## If true, enables cluster-wide webhooks.
+    #
+    enabled: false
+
+    ## @param namespaceSelector - LabelSelector - optional
+    ## NamespaceSelector decides whether to run the webhook on an object based on whether the
+    ## namespace for that object matches the selector.
+    ## The matching is performed on object.metadata.labels.
+    ## Default to the empty LabelSelector, which matches everything.
+    ##
+    ## ref: https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/#set-references-in-api-objects
+    ## ref: https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.17/#mutatingwebhook-v1beta1-admissionregistration-k8s-io
+    ## ref: https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.17/#labelselector-v1-meta
+    #
+    # namespaceSelector:
```

Comment:
Is this an option that will be used? Just want to make sure we don't add options that will never be used and that make the config harder to understand.

Reply:
The ability to select what pods the webhook modify seem to me like a very important configuration, given the default scope of matching every single pod.

#containers would be best to chime in, but I would wager it is a useful setting.